doc/source/whatsnew/v1.1.0.rst

```diff
@@ -941,7 +941,7 @@ Timezones
 ^^^^^^^^^
 
 - Bug in :func:`to_datetime` with ``infer_datetime_format=True`` where timezone names (e.g. ``UTC``) would not be parsed correctly (:issue:`33133`)
--
+- Bug in :func:`date_range` was raising AmbiguousTimeError for valid input with `ambiguous=False` (:issue:`35297`)
```

Comment:
Could you move this to 1.2.0?

Reply:
For sure! I don't think we have `doc/source/whatsnew/v1.2.0.rst` atm - shall I create it?

Reply:
Sure that'd be great.